frontend/src/components/GoesData/BrowseTab.tsx

```
@@ -436,349 +84,103 @@ export default function BrowseTab() {
     <div className="space-y-2">
-      {/* Mobile filter toggle — own row above the main flex layout */}
       <div className="md:hidden flex justify-end">
-        <button
-          onClick={() => setShowBottomSheet(true)}
+        <button onClick={() => setShowBottomSheet(true)}
           className="flex items-center gap-2 px-3 py-2 rounded-lg bg-gray-100 dark:bg-slate-800 text-sm font-medium text-gray-600 dark:text-slate-300 min-h-[44px]"
-          aria-label="Toggle filters"
-        >
-          <SlidersHorizontal className="w-4 h-4" />
-          Filters
+          aria-label="Toggle filters">
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)